ktor-client/ktor-client-core/common/test/HttpClientConfigTest.kt

```diff
@@ -0,0 +1,44 @@
+import io.ktor.client.*
+import io.ktor.client.plugins.api.*
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+/*
+ * Copyright 2014-2024 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
```

Comment:
IDEA adds imports above copyright note if a package is not specified. Could you move copyright to the top?

Reply:
sure!